README.md

```diff
@@ -1,6 +1,13 @@
 Haskell Platform
 ----------------
 
+"I WANT TO INSTALL HASKELL"
```

Comment:
small nit but I think leaving the existing readme in place and adding a deprecation warning clearly above it like ahttps://github.com/haskell/ghcup does is slightly clearer

Reply:
I'm happy to go with either.